include/shellstrings.h

```diff
@@ -19,7 +31,7 @@
  * Returns:
  *  - None.
  */
-void shell_prompt();
+void shell_prompt(bool *color);
```

Comment:
Why is this a `bool *` and not just a `bool`?

Reply:
changed!